Formula/jenkins.rb

```diff
@@ -1,8 +1,8 @@
 class Jenkins < Formula
   desc "Extendable open source continuous integration server"
   homepage "https://jenkins-ci.org"
-  url "http://mirrors.jenkins-ci.org/war/1.655/jenkins.war"
-  sha256 "0cee889af697c115961ce50229cc5e39d1b798c0a0a689687b745c0a938c8547"
+  url "http://mirrors.jenkins-ci.org/war/1.656/jenkins.war"
+  sha256 "97085359725b16aca0f419486e5c1809399e33dbe6d906cd55ef82fad76fb30d"
 
   devel do
     url "http://mirrors.jenkins-ci.org/war-rc/2.0/jenkins.war"
```

Comment:
Checksum for this looks wrong: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/643/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/fetch___devel_jenkins/

Looks like this is something that isn't pinned to a specific revision so probably needs removed.